.github/workflows/main.yml

```
@@ -92,3 +92,25 @@ jobs:
       # The Windows CI transforms the `c-library` symlink into a real directory, modifying the Git
       # state, so we ignore these changes with `--allow-dirty` here.
       run: scripts/verify-publish.sh --allow-dirty
+
+  itt_python_build_and_test:
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - os: ubuntu-latest
+          - os: windows-latest
+    runs-on: ['${{matrix.os}}']
+
+    steps:
+      - name: Checkout code
```

```suggestion
      - name: Checkout sources
```
use the same wording as in existing jobs